gatsby-config.js

```diff
@@ -13,16 +13,13 @@ module.exports = {
     {
       resolve: "gatsby-plugin-manifest",
       options: {
-        icon: "node_modules/@jetbrains/kotlin-web-site-ui/dist/svg/kotlin_64.svg",
+        icon: "node_modules/@jetbrains/kotlin-web-site-ui/out/svg/kotlin_64.svg",
       },
     },
     {
       resolve: "gatsby-plugin-mdx",
       options: {
         extensions: [ '.md' ],
-        defaultLayouts: {
```

Comment:
Function was [dropped](https://www.gatsbyjs.com/plugins/gatsby-plugin-mdx/#layouts) :(

Reply:
Ooh, okey :( 
Could you please add a remark to the "How to Add a Page" section in the Readme.md about adding the layout import?